modules/KalturaSupport/resources/mw.ClosedCaptions.js

```diff
@@ -115,6 +115,7 @@
 						_this.destory();
 						_this.setupTextSources( function () {
 							_this.buildMenu( _this.textSources );
+							_this.unbind('playerReady');
```

Comment:
@GuyLivni, @itaykinnrot 
This fix broke use case of  change-media + forcedCaptionLoaded. We MUST take it out or fix this issue on another approach. With this version of code any 2nd media is not dispatching forcedCaptionLoaded hence 3rd party plugins cannot know that a new caption file was loaded.